### PR TITLE
ign -> gz Upstream Macro Migration : gz-gui

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM once upstream PRs are merged and CI is happy 🟢

File: src/Plugin.cc [hunk 1]
```diff
@@ -167,26 +167,45 @@ void Plugin::Load(const tinyxml2::XMLElement *_pluginElem)
...
+  {
+    guiElem = _pluginElem->FirstChildElement("ignition-gui");
+    if (guiElem)
+    {
+      gzwarn << "The `ignition-gui` element is deprecated. Please use "
+      << "`gz-gui` instead." << std::endl;
+    }
+    else
+    {
+      this->LoadCommonConfig(_pluginElem->FirstChildElement("gz-gui"));
```
Comment (chapulina) [inline src/Plugin.cc:187]: Should be up in the `if` together with the warning, and say `ignition-gui`?

File: src/Plugin.cc [hunk 2]
```diff
@@ -290,24 +309,35 @@ std::string Plugin::ConfigStr()
...
-  // <ignition-gui>
-  auto ignGuiElem = pluginElem->FirstChildElement("ignition-gui");
-  if (!ignGuiElem)
+  // <gz-gui>
+  auto guiElem = pluginElem->FirstChildElement("gz-gui");
+  if (!guiElem)
   {
-    ignGuiElem = doc.NewElement("ignition-gui");
-    pluginElem->InsertEndChild(ignGuiElem);
+    // TODO(CH3): Deprecated. Remove on tock.
```
Comment (chapulina) [inline src/Plugin.cc:320]: This function is used for saving the config, so I think just changing `ignition` to `gz` is enough. No need to tick-tock. All new saved configs will have `gz`, which is ok. The tick-tock you did for loading old configs that contain `ignition` is enough to tick-tock this feature.

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/Plugin.cc (hunk 2 above)
Comment (methylDragon) [reply src/Plugin.cc]: https://github.com/gazebosim/gz-gui/pull/413/commits/fe598592f38834a889c2d6bde41bcaee30b3c1d5 un-tick-tocked!

**Comment by chapulina** [comment]
@osrf-jenkins run tests please!

**Pull request merged**